catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/integration/test/CatalogIcebergIT.java

```diff
@@ -1051,6 +1052,114 @@ public void testTableDistribution() {
             "Iceberg's Distribution Mode.RANGE not support set expressions."));
   }
 
+  @Test
+  void testIcebergTablePropertiesWhenCreate() {
+    // Create table from Gravitino API
+    Column[] columns = createColumns();
+
+    NameIdentifier tableIdentifier =
+        NameIdentifier.of(metalakeName, catalogName, schemaName, tableName);
+    Distribution distribution = Distributions.NONE;
+
+    final SortOrder[] sortOrders =
+        new SortOrder[] {
+          SortOrders.of(
+              NamedReference.field(ICEBERG_COL_NAME2),
+              SortDirection.DESCENDING,
+              NullOrdering.NULLS_FIRST)
+        };
+
+    Transform[] partitioning = new Transform[] {Transforms.day(columns[1].name())};
+    Map<String, String> properties = createProperties();
+    TableCatalog tableCatalog = catalog.asTableCatalog();
+    Table createdTable =
+        tableCatalog.createTable(
+            tableIdentifier,
+            columns,
+            table_comment,
+            properties,
+            partitioning,
+            distribution,
+            sortOrders);
+    Assertions.assertFalse(createdTable.properties().containsKey(DEFAULT_FILE_FORMAT));
+    Table loadTable = tableCatalog.loadTable(tableIdentifier);
+    Assertions.assertFalse(loadTable.properties().containsKey(DEFAULT_FILE_FORMAT));
+
+    tableCatalog.dropTable(tableIdentifier);
+    properties.put("provider", "iceberg");
+    createdTable =
+        tableCatalog.createTable(
+            tableIdentifier,
+            columns,
+            table_comment,
+            properties,
+            partitioning,
+            distribution,
+            sortOrders);
+    Assertions.assertFalse(createdTable.properties().containsKey(DEFAULT_FILE_FORMAT));
+    loadTable = tableCatalog.loadTable(tableIdentifier);
+    Assertions.assertFalse(loadTable.properties().containsKey(DEFAULT_FILE_FORMAT));
+
+    tableCatalog.dropTable(tableIdentifier);
```

Comment:
1. use `ICEBERG_ORC_FILE_FORMAT ` not `orc` directly?
2. do some abstraction to simple the code?

Reply:
ok.

Reply:
updated.